app/res/values/strings.xml

```diff
@@ -935,4 +935,11 @@
     <string name="camera_permission_title">Permission for camera</string>
     <string name="camera_permission_msg">In order to take a picture, CommCare needs permission to use your device camera.</string>
     <string name="capture_photo">Capture Photo</string>
+    <string name="incorrect_otp">The OTP entered is incorrect. Please try again.</string>
+    <string name="too_many_attempts">Too many attempts. Please try again later.</string>
+    <string name="missing_activity">Verification failed due to missing activity context. Please retry.</string>
```

Comment:
When will this happen ? 

Also We can't show this error to users and should just say more on the lines of  "Something went wrong"